mmcls/models/losses/accuracy.py

```diff
@@ -56,6 +56,7 @@ def accuracy_torch(pred, target, topk=(1, ), thrs=0.):
     res = []
     maxk = max(topk)
     num = pred.size(0)
+    pred = pred.float()
```

Comment:
Is this line necessary?

Reply:
when train in fp16, pred will be half, and torch cpu not implement topk on half